src/module_TOUCHSCREEN/SerialComm.h

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
I noticed this is currently in the TOUCHSCREEN Module's directory. Is that the best place for this? It seems like it is more related to a PI module than specific to the TOUCHSCREEN.